js/notify.js

```diff
@@ -15,7 +15,7 @@ define([
             level: type == 'error' ? 'danger' : type,
             message: message
         }));
-        Log.debug('Notify:', {type: type, message: message, container: container});
+        Log.default.debug('Notify:', {type: type, message: message, container: container});
```

Comment:
I'm not sure why we have a _default_ property here that access the functions yet it works